common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/WebViewAuthorizationFragment.java

```diff
@@ -281,6 +282,17 @@ public void onPermissionRequest(final PermissionRequest request) {
                 }
 
             }
+
+            @Override
+            public Bitmap getDefaultVideoPoster() {
+                // When not playing, video elements are represented by a 'poster' image.
+                // The image to use can be specified by the poster attribute of the video tag in HTML.
+                // If the attribute is absent, then a default poster will be used.
+                // This method allows the ChromeClient to provide that default image.
+                // On our case when an invalid QR code is provided,  we will return a 10x10 empty image.
+                // Instead of the default grey playback image. #2424
+                return Bitmap.createBitmap(10, 10, Bitmap.Config.ARGB_8888);
```

Comment:
TBH, the empty whitespace as you've shown in the AFTER screenshot looks weird too.

Reply:
Rather than returning an empty image, can we work with PM to create a dedicated ERROR image that renders some meaningful error img/text such as `Oops something went wrong :(` and then just use that?

Reply:
Waiting for PM to decide what they want to do here.

Reply:
ESTS UX made the fix and now the error page changed.  but still, we need to remove this image because this image is shown for a second before loading the camera. which looks odd.